packages/web-react/tests/providerTests/validationTextPropsTest.tsx

```diff
@@ -38,7 +38,8 @@ export const validationTextPropsTest = (
 
     await waitFor(() => {
       const element = dom.container.querySelector(selector) as HTMLElement;
-      expect(element.tagName.toLowerCase()).toBe('ul');
+      expect(element.tagName.toLowerCase()).toBe('div');
```

Comment:
**suggestion (if-minor):** What about refactoring this using `screen` and aria roles like we do it in other tests?

Reply:
will do 👍 , i am still working on the react package so there might be some changes in the next commits

Reply:
After consulting with Kuba, we decided to leave these tests as they are